community_images/consul/bitnami/scripts/server_coverage_script.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+set -x
+set -e
+
+# Available Scripts
+ls /opt/bitnami/scripts
+
+# consul acl token list
+# consul acl boostrap
+consul members
+consul info
+
+# Registering a test service
+consul services register /consul.d/sample_service.json
+consul reload
+sleep 10
+
+# Query our service using HTTP Api
+curl http://localhost:8500/vi/catalog/service/web
+
+# Checking for the healthy instances
+curl 'http://localhost:8500/v1/health/service/web?passing'
```

Comment:
add a verification on the number of instances as well

Reply:
Could you please guide me how?

Reply:
I meant the number of members in the consul cluster we can verify to be 3 in our case

Reply:
3 servers and 1 client, yes. Added.